installing/disconnected_install/installing-mirroring-disconnected-v2.adoc

```diff
@@ -0,0 +1,305 @@
+====
+
+[id="using-oc-mirror"]
+== Mirroring an image set to a mirror registry
+
+You can use this workflow when the target disconnected registry is fully disconnected. In other words, the environment on which you are executing oc-mirror has access to the internet but doesn't have access to the target disconnected registry.
+
+.Environment Preparation
+
+* To set up the credentials for pulling/pushing container images, download the pull secret and place it in the correct path.
+
+** The default podman credentials location `$XDG_RUNTIME_DIR/containers/auth` is used for authenticating to the registries.
+** The docker location `~/.docker/config.json` for credentials is also supported.
+** Save the file as `$XDG_RUNTIME_DIR/containers/auth.json`.
+
+[id="building-image-set-configuration"]
+=== Building the image set configuration
+
+The Image Set Configuration is the input file used by oc-mirror to discover which images are necessary to be mirrored.
+
+Below is an example for the `ImageSetConfiguration` input file:
+
+[source,yaml]
+----
+kind: ImageSetConfiguration
+apiVersion: mirror.openshift.io/v2alpha1
+mirror:
+  platform:
+    channels:
+    - name: stable-4.13
+      minVersion: 4.13.10
+      maxVersion: 4.13.10
+    graph: true
+  operators:
+    - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.15
+      packages:
+       - name: aws-load-balancer-operator
+       - name: 3scale-operator
+       - name: node-observability-operator
+  additionalImages: 
+   - name: registry.redhat.io/ubi8/ubi:latest
+   - name: registry.redhat.io/ubi9/ubi@sha256:20f695d2a91352d4eaa25107535126727b5945bff38ed36a3e59590f495046f0
+----
+
+[id="oc-mirror-workflows-partially-disconnected"]
+=== Mirroring an image set in a partially disconnected environment
+
+.Mirror To Mirror
+
+You can use this workflow for partially disconnected mirroring.
+You can use this workflow when the environment from which you are executing oc-mirror has both:
+
+- Access to the internet
+- Access to the target disconnected registry
+
+You are required to specify a storage backend in the image set configuration file. This storage backend can be a local directory or a Docker v2 registry. The oc-mirror plugin stores metadata in this storage backend during image set creation.
+
+.Prerequisites
+
+* You have access to the internet to get the necessary container images.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the `oc-mirror` CLI plugin.
+* You have created the image set configuration file.
+
+.Procedure
+
+Copy the container images from the source specified in the image set configuration to the destination (container registry).
+
+[source,terminal]
+----
+$ oc mirror -c isc.yaml --workspace file://oc-mirror/mirror1 docker://localhost:6000 --v2
+----
+
+.Verification
+
+. Navigate into the `oc-mirror-workspace/` directory that was generated.
```

Comment:
Instead of `oc-mirror-workspace` it is called `working-dir` under the path that you specified on --workspace